src/style.css

```
@@ -0,0 +1,71 @@
+body {
+  margin: 0;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+  
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  max-width: 1199px;
+  
+}
+
+.block {
+  flex-basis: 300px;
+  flex-grow: 1;
+  text-align: center;
+  line-height: 300px;
+  
+}
+
+.block-1 {
+  background-color: red;
+  order: 1;
+}
+
+.block-2 {
+  background-color:rgb(200,0,0);
+  order: 2;
+}
+
+.block-3 {
+  background-color:rgb(150,0,0);
+  order: 3;
+}
+
+.block-4 {
+  background-color:rgb(100,0,0);
+  order: 4;
+}
+
+.block-5 {
+  background-color:rgb(50,0,0);
+  order: 5;
+}
+
+.block-6 {
+  background-color:black;
+  order: 6;
+}
+
+@media (min-width: 600px) and (max-width:899px) {
```

```suggestion
@media (min-width: 600px) {
```
Нижний медиа запрос перекроет стили этого, для block-4, можно не писать and